libs/langchain-mistralai/src/chat_models.ts

```diff
@@ -531,79 +531,45 @@ export class ChatMistralAI<
   }
 
   withStructuredOutput<
-    RunInput = BaseLanguageModelInput,
-    // prettier-ignore
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    RunOutput extends z.ZodObject<any, any, any, any> = z.ZodObject<any, any, any, any>
+    RunOutput extends Record<string, any> = Record<string, any>
```

Comment:
will this still allow for zod schemas to be passed in?

Reply:
No